build.lua

```diff
@@ -7,6 +7,16 @@ bundle = ""
 -- Non-standard settings
 checkconfigs = {"build", "config-pdf", "config-plain"}
 checkdeps    = { }
+--[[ FIXME:
+  - setting specialformats in config-pdf.lua results in "binary" set to "latexdvips" (should be "latex")
+  - setting specialformats in build.lua disables specifying engine
+      l3build save -c config-pdf -e latexdvips 00-test-2
+]]
+specialformats = specialformats or {}
+specialformats["latex"] = specialformats["latex"] or
+  {
+    latexdvips = {binary = "latex", format = ""}
+  }
```

Comment:
I guess the wired limitations are caused by some `l3build` problems, but for now let's just use a way that "works".